src/blocks/CodeFeature/index.module.scss

```diff
@@ -1,16 +1,31 @@
 @use '@scss/common' as *;
 
-.codeFeature {
-  position: relative;
-}
+.wrapper {
+  &.darkTheme {
+    background: var(--color-base-950);
 
-.blockSpacing {
-  padding-top: var(--block-spacing);
-  padding-bottom: var(--block-spacing);
+    & .backgroundGrid {
+      z-index: 0;
+    }
+  }
 }
 
-.codeFeatureGrid {
+.container {
+  position: relative;
   align-items: center;
+  row-gap: calc(var(--base) * 3);
```

Comment:
@paulpopus I would reduce the `row gap` in `mid` and `small` breaks. Seems like too much spacing in mobile

![Screen Shot 2024-02-05 at 11 55 04 AM](https://github.com/payloadcms/website/assets/35232443/972f38f0-be5a-4b48-ab16-8525d07ca2fd)